src/SprykerEco/Zed/PunchoutCatalogs/Business/Writer/PunchoutCatalogsWriter.php

```diff
@@ -10,25 +10,35 @@
 use Generated\Shared\Transfer\MessageTransfer;
 use Generated\Shared\Transfer\PunchoutCatalogConnectionTransfer;
 use Generated\Shared\Transfer\PunchoutCatalogResponseTransfer;
+use SprykerEco\Zed\PunchoutCatalogs\Dependency\Facade\PunchoutCatalogsToVaultFacadeInterface;
 use SprykerEco\Zed\PunchoutCatalogs\Persistence\PunchoutCatalogsEntityManagerInterface;
 
 class PunchoutCatalogsWriter implements PunchoutCatalogsWriterInterface
 {
     protected const MESSAGE_ERROR_DURING_CONNECTION_UPDATE = 'Error during connection update';
     protected const MESSAGE_ERROR_DURING_CONNECTION_CREATION = 'Error during connection creation';
+    protected const VAULT_DATA_TYPE_PASSWORD = 'pwg_punchout_catalog_connection.password';
```

Comment:
`VAULT_DATA_TYPE_CATALOG_CONNECTION_PASSWORD`?

Reply:
This one is better thanks, but I would use 

CATALOG_CONNECTION_PASSWORD_VAULT_DATA_TYPE

Because it's about data type for password, but not a password for data type.